pkg/controlplane/manager.go

```diff
@@ -150,7 +150,10 @@ func (manager *Manager) isEndpointAccessible() bool {
 func (manager *Manager) isKubeletServiceRunning() bool {
 	url := fmt.Sprintf("https://%s:%s/pods", manager.nodeName, kubeletPort)
 	tr := &http.Transport{
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+		TLSClientConfig: &tls.Config{
+			InsecureSkipVerify: true,
+			MinVersion:         tls.VersionTLS13,
```

Comment:
can we use the const here as well ?

Reply:
I wonder if `credentials` is the right package where to store this. Maybe a new file under `utils`?

Reply:
IMO `credentials` is a decent fit since this field has to do with `credentials`.
`utils` is more general so usually I'd try to avoid it when a better fit is possible.
Not a deal breaker as far as I'm concerned, so  if you strongly feel different that's fine too.

Reply:
I agree with `credentials`